File: pgm/ABC_Java/ABC_306_A.java
import java.io.*;
// import java.math.*;
import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final int N = Integer.parseInt (reader.readLine());
        final String[] S = reader.readLine().split ("");
        final int two = 2;
        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < N; i++) {
            sb.append (String.join ("", Collections.nCopies (two, S[i])));
        }
        System.out.println (sb.toString());
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_306_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.StringTokenizer

// import kotlin.math.*

const val blank: String = ""
const val two: Int = 2

fun main() {
    readln()
    val S: String = readln()
    println(S.map { c: Char ->
        c.toString().repeat(two)
    }.joinToString(separator = blank))
    kotlin.system.exitProcess(0)
}